lib/linter-rust.coffee

```diff
@@ -49,6 +55,9 @@ class LinterRust
         else
           resolve []
 
+      if do @ableToJSONErrors
```

Comment:
Since this is now being called _three_ times for every lint call, it's results should probably be cached...